src/TorchSharp/Tensor/TensorExtensionMethods.cs

```diff
@@ -335,5 +340,28 @@ public static double clip_grad_norm(this IList<TorchTensor> tensors, double max_
 
         public static (float Real, float Imaginary) ToComplex32(this TorchTensor value) => value.ToScalar().ToComplexFloat32();
         public static System.Numerics.Complex ToComplex64(this TorchTensor value) => value.ToScalar().ToComplexFloat64();
+
+        // Vision-related operations
+
+        public static TorchTensor crop(this TorchTensor image, int top, int left, int height, int width)
+        {
+            var dims = image.Dimensions;
+            var hoffset = dims - 2;
+            long h = image.shape[hoffset], w = image.shape[hoffset + 1];
+
+            var right = left + width;
+            var bottom = top + height;
+
+            if (left < 0 || top < 0 || right > w || bottom > h) {
+
+                //var padding_ltrb = new long[] { Math.Max(-left, 0), Math.Max(-top, 0), Math.Max(right - w, 0), Math.Max(bottom - h, 0) };
```

Comment:
Add a comment on why this code is commented out?

Reply:
Yes.